client/delegations.go

```diff
@@ -39,11 +39,11 @@ func (c *Client) getTargetFileMeta(target string) (data.TargetFileMeta, error) {
 		}
 
 		if targets.Delegations != nil {
-			delegationsVerifier, err := verify.NewDelegationsVerifier(targets.Delegations)
+			delegationsDB, err := verify.NewDBFromDelegations(targets.Delegations)
```

Comment:
probably the old names are still good regardless of the type returned here?

Reply:
True, but `delegationsDB` is both shorter and more accurate.

Reply:
That should have been even more confusing. But now that this is actually a mapping to keys, using DB does not make sense anymore an confusing, specifically that there is a db per delegation path (is my understanding correct?)